src/axom/quest/detail/MarchingCubesFullParallel.hpp

```diff
@@ -0,0 +1,897 @@
+// Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include "axom/config.hpp"
+
+// Implementation requires Conduit.
+#ifdef AXOM_USE_CONDUIT
```

Comment:
Do we need to guard this file by `AXOM_USE_CONDUIT` if the build system only adds it in configurations w/ conduit?

Consider converting this `#if` guard to the following check at the top of the file:
```cpp
#ifndef AXOM_USE_CONDUIT
  #error "MarchingCubesFullParallel.hpp requires conduit"
#endif
```

Reply:
Will do.